simphony_metadata.yml

```diff
@@ -13,41 +13,41 @@ Resources:
 CUDS_KEYS:
 
   CUDS_ITEM:
-    parent: 
+    parent:
     CUBA.UUID:
       scope: CUBA.SYSTEM
-    CUBA.DATA:  
-      
+    CUBA.DATA:
+
   CUDS_COMPONENT:
     definition: Base data type for the CUDS components
     parent: CUBA.CUDS_ITEM
     CUBA.DESCRIPTION:
```

Comment:
Looks to me that  `CUBA.DESCRIPTION` is not defined in the cuba.yml
And further down the file there are "description: " (e.g. https://github.com/simphony/simphony-metadata/pull/7/files#diff-ee11a1e13dd98ffdcd89c6b550a3bc71R343), should they be `CUBA.DESCRIPTION` too?